samples/excel/16-custom-functions/volatile-function.yaml

```diff
@@ -7,18 +7,14 @@ api_set:
     CustomFunctionsRuntime: 1.1
 script:
     content: |
-        /** @CustomFunction */
-        function ROLL6SIDEDICE(x: number): number {
-          let i:number;
-          let result:number;
-          result = 0;
-          for (i = 0;i<x;i++) {
-            var roll:number;
-            roll = Math.random()*6;
-            roll = Math.round(roll);
-            result += roll;
-          }
-          return result;
+        /** @CustomFunction 
+        *  @description Simulates rolling a 6 sided die.
+        * @returns A whole number from 1 to 6.
+        * @volatile
+        */
+        function roll6sided(): number {
+          const roll: number = Math.random() * 5 + 1;
```

Comment:
Nit: we usually don't explicitly define the type in our samples.

Reply:
+1.  Typescript will figure it out, so you don't need it.

In fact, you could just simplify it to `return Math.round(Math.random() * 5 + 1)` as a one-liner, and I think it would still be perfectly readable

Reply:
I like the one liner. Changed!